src/create-element.js

```diff
@@ -93,6 +93,10 @@ export function Fragment(props) {
 	return props.children;
 }
 
+export function Root(props) {
```

Comment:
What's the reason for having the separate constructor here, versus just reusing Fragment?

If possible, I would really like to avoid exporting `Root` since it's an implementation detail. I think there might be a way to use a signal property no a Fragment vnode.